sdl_viewer/src/terrain_drawer/read_write.rs

```diff
@@ -0,0 +1,56 @@
+use crate::graphic::tiled_texture_loader::{TilePos, TiledTextureLoader};
+use cgmath::Vector3;
+use image::{LumaA, Rgba};
+use point_viewer::math::Isometry3;
+use serde::{Deserialize, Serialize};
+use std::fs::File;
+use std::io::{Error, ErrorKind};
+
+#[derive(Serialize, Deserialize)]
+pub struct Metadata {
+    pub tile_size: u32,
+    pub world_from_terrain: Isometry3<f64>,
+    pub origin: Vector3<f64>,
+    pub resolution_m: f64,
+    pub tile_positions: Vec<TilePos>,
+}
+
+pub type TextureLoaders = (TiledTextureLoader<LumaA<f32>>, TiledTextureLoader<Rgba<u8>>);
+
+#[allow(dead_code)]
```

Comment:
Do we need this attribute?

Reply:
Yeah, for now. Will be removed in the next PR.